README.adoc

```diff
@@ -6,7 +6,6 @@
 image:https://img.shields.io/jenkins/plugin/v/gradle.svg[Jenkins Plugin,link=https://plugins.jenkins.io/gradle]
 image:https://img.shields.io/github/release/jenkinsci/gradle-plugin.svg?label=changelog[link="https://github.com/jenkinsci/gradle-plugin/releases/latest"]
 image:https://img.shields.io/jenkins/plugin/i/gradle.svg?color=blue[Jenkins Plugin Installs,link=https://plugins.jenkins.io/gradle]
-image:https://ci.jenkins.io/buildStatus/icon?job=Plugins/gradle-plugin/master[Build Status,link=https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgradle-plugin/branches/]
```

Comment:
Why remove the link to the Jenkins build here?

Reply:
Because it has been red most of the time in the past due to the flaky browser tests. We can keep it but having it red sends out a wrong signal and therefore I'd rather remove it.

Reply:
Alright, if you prefer not having the link at all that is fine with me.

Reply:
@guylabs I think we need to keep it and just move browser tests to TeamCity and don't run them on Jenkins. Tests are flaky due to the infra, on TeamCity the tests should be stable because we have control over the infrastructure.

Reply:
@c00ler what kind of issues have you encountered on Jenkins infrastructure? Is there something we could do?

Reply:
Hi @lemeurherve, thank you for looking into it! 
We are running many browser tests, with different versions of Jenkins, and were facing some performance constraints, due to the agent machines' size. But this is something definitely expected. 
Another issue that we've seen quite frequently are failures due to the browser driver, for example [here](https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgradle-plugin/detail/master/181/tests), or [here](https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fgradle-plugin/detail/PR-210/2/tests/). It's hard to say at the moment what's the cause of these issues and if anything can be done to improve the stability.

Reply:
@c00ler thanks for the feedback :)

About the agent size, you can use the beefier [highmem](https://github.com/jenkins-infra/jenkins-infra/blob/93f7c1536b726d95cc9e6dec78779e56bad323c0/hieradata/clients/azure.ci.jenkins.io.yaml#L322) node label for your browser tests.

For the failures you encountered, would you mind [opening an help desk issue](https://github.com/jenkins-infra/helpdesk) so we can take a more closer look at them please?

Reply:
I realized you can't choose the label with buildPluginWithGradle, I'll implement it so you can set an `highmem` label.